src/main/java/com/solvd/carina_testing/components/PopUpWindowBag.java

```diff
@@ -0,0 +1,28 @@
+package com.solvd.carina_testing.components;
+
+import com.solvd.carina_testing.pages.CheckoutPage;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
+import jdk.dynalink.linker.LinkerServices;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class PopUpWindowBag extends AbstractUIObject {
+
+    @FindBy(xpath = "//button[contains(text(), 'Зробити замовлення')]")
+    private ExtendedWebElement buttonCreateOrder;
+    public PopUpWindowBag(WebDriver driver, SearchContext searchContext) {
+        super(driver, searchContext);
+    }
+
+    public ExtendedWebElement getButtonCreateOrder() {
```

Comment:
usually we're trying to avoid methods with returning ExtendedWebElement. If you need getButtonCreateOrder just for checking if button present, It's much better to create **public boolean isCreateOrderButtonPresent()** method